include/ofi_prov.h

```diff
@@ -304,6 +304,21 @@ UCX_INI ;
 #  define UCX_INIT NULL
 #endif
 
+#if defined(_WIN32) && (HAVE_EFA)
```

Comment:
Is LPP somewhat related to EFA, or is it a typo?

Reply:
That's a typo.